core/designsystem/src/androidMain/kotlin/io/github/droidkaigi/confsched2023/designsystem/preview/Annotations.kt

```diff
@@ -60,25 +60,29 @@ object MultiLanguagePreviewDefinition {
     object Japanese {
         const val Name = "Japanese"
         const val Locale = "ja_JP"
+        const val LocaleShort = "ja"
     }
 
     object English {
         const val Name = "English"
         const val Locale = "en_US"
+        const val LocaleShort = "en"
```

Comment:
@takahirom 
Thank you for your comment :pray: 

I was wondering what the names are good for these.
**Engligh/Japanese** object means language name, and `Name` variable also has language info.
And `java.util.Local` has `Locale.JAPAN` (it means "ja_JP") and `Locale.JAPANESE` (it means "ja") or `Locale.ENGLISH` ("en") and `Locale.US` ("en_US").
So we also can consider that both type of values are `Locale`.
I choiced `LocaleShort` because I felt `Language` is little bit confusing with other value (object name or `Name`). **(IMO)**

But if `Language` is natural for you, I think it's fine too!

I modified about this!! (https://github.com/DroidKaigi/conference-app-2023/pull/942/commits/1efd7b2a34d267557187f9c4dca5947537c34461)